### PR TITLE
[ContextualMenu] Render persistent contextual menus with correct scrollbar in Edge.

### LINKED ISSUE #9034: Contextual menu scroll bar is hiding right arrow
### Environment Information 
 
- __Package version(s)__: Latest
- __Browser and OS versions__: Edge

### Please provide a reproduction of the bug in a codepen:
 
https://developer.microsoft.com/en-us/fabric#/controls/web/contextualmenu > example "ContextualMenu which is persisted in the DOM" or in outlook web app right click on a mail list item.
 
#### Actual behavior:
 
The scroll bar hides the right arrow to see submenu items.

image
 
#### Expected behavior:
 
The scroll bar does not hide the right arrow.

image

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? Maybe
 
Requested priority: Normal
 
Products/sites affected: Outlook web app (and anyone else who has long context menus and/or users with small browser window heights)

### PR DESCRIPTION
**Pull request by jdhuntington**
[ContextualMenu] Render persistent contextual menus with correct scrollbar in Edge.

#### Pull request checklist

- [X] Addresses an existing issue: Fixes #9034
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Causes persistent menus to be persistent only after they have rendered 1 time. In Edge, menus rendered while hidden had incorrect scrollbar behavior.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9124)

### CODE REVIEW

**Comment by joschect** [comment]
Only suggestion, make sure that we open it when we take a snap to preserve those.

File: common/changes/office-ui-fabric-react/fix-contextual-menu-scrollbar_2019-05-16-21-22.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "[ContextualMenu] Render persistent contextual menus with correct scrollbar in Edge.",
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/fix-contextual-menu-scrollbar_2019-05-16-21-22.json]: (fwiw: same comment as in your other PR :P)

File: packages/office-ui-fabric-react/src/components/__snapshots__/DetailsList.Advanced.Example.tsx.shot [hunk 2]
```diff
@@ -4,7 +4,7 @@ exports[`Component Examples renders DetailsList.Advanced.Example.tsx correctly 1
 <div
   className=
 
-      & .headerDivider-479:hover + .headerDividerBar-480 {
+      & .headerDivider-449:hover + .headerDividerBar-450 {
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/__snapshots__/DetailsList.Advanced.Example.tsx.shot]: um. What? these shoudl not be here. Something is very wrong.

File: common/changes/office-ui-fabric-react/fix-contextual-menu-scrollbar_2019-05-16-21-22.json (hunk 1 above)
Comment (dzearing) [reply common/changes/office-ui-fabric-react/fix-contextual-menu-scrollbar_2019-05-16-21-22.json]: Yeah we have this format:

ContextualMenu: <what's different that the user can expect.>

These go into the release notes, so you should think about how the user will read them and understand what behavior is different. Example:

ContextualMenu: When menus are rendered with the `persistMenu` flag, the scrollbar now renders correctly in Edge.

Comment (dzearing) [reply common/changes/office-ui-fabric-react/fix-contextual-menu-scrollbar_2019-05-16-21-22.json]: you can use backticks and general markdown in the string as needed.

File: packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.test.tsx [hunk 3]
```diff
@@ -918,6 +919,48 @@ describe('ContextualMenu', () => {
     expect(customRenderer).toHaveBeenCalledTimes(2);
   });
 
+  describe('ContextualMenu snapshot', () => {
+    it('ContextualMenu should be present in DOM when hidden', () => {
```
Comment (khmakoto) [inline packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.test.tsx]: Suggestion: I see that this test has the same name as the one from line 1004. Is there some differentiation we can make in the naming?

**Pull request merged**